src/lib/gundb/__tests__/UserStorage.js

```diff
@@ -627,14 +632,25 @@ describe('UserStorage', () => {
   it(`update profile doesn't change privacy settings`, async done => {
     const email = 'johndoe@blah.com'
     await userStorage.setProfileField('email', email, 'public')
-    await userStorage.setProfile(getUserModel({ email, fullName: 'full name', mobile: '+22222222222' }))
+    await userStorage.setProfile(getUserModel({ email, fullName: 'full name', mobile: '+22222222222' }), true)
     userStorage.subscribeProfileUpdates(updatedProfile => {
       const result = userStorage.getDisplayProfile(updatedProfile)
       expect(result.email).toBe(email)
       done()
     })
   })
 
+  it(`setting profile changes privacy settings`, async done => {
+    const email = 'johndoe@blah.com'
+    await userStorage.setProfileField('email', email, 'public')
+    await userStorage.setProfile(getUserModel({ email, fullName: 'full name', mobile: '+22222222222' }))
+    userStorage.subscribeProfileUpdates(updatedProfile => {
```

Comment:
need to call unsubscribe on the test's tearDown

Reply:
there's an unsubscribe defined in the begining afterEach